README.md

```diff
@@ -1,2 +1,44 @@
 # Blog
 The Blog app will be a classic example of a blog website. I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## Built With
+
+- Visual Studio Code and GitHub.
+- Ruby on Rails.
+- Linters for Ruby on Rails.
+- Gitflow.
+
+## Getting Started
+
+I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
```

Comment:
- [OPTIONAL] Please it will be nice to add a `setup section` with ` explicit steps` on how to clone and run your app locally, as this will be of help to none technical users who would like to know how your app works.

Reply:
Thank you @t-yanick I will add it.